src/templates/post.js

```diff
@@ -12,49 +12,26 @@ import SignUp from "../components/SignUp"
 import sendToGoogleAnalytics from "../utils/vitals"
 import "../pages/styles.scss"
 
-let embedRegex = /\[embed src=\s*(.*?)\]/i;
+// Look for URLs in the article copy for embedding social media
+let urlRegex = /(https?:\/\/)?([\w\-])+\.{1}([a-zA-Z]{2,63})([\/\w-]*)*\/?\??([^#\n\r]*)?#?([^\n\r]*)/i; 
```

Comment:
Should this regex only match on embeds we support? Youtube, Twitter, etc.

Reply:
Good point. I'll update it.

Reply:
the article template (which I really should rename at some point) now has a lot of regular expressions for testing if a URL is on a supported platform. right now that includes:

* dailymotion
* facebook
* google 
* imgur
* instagram
* mixcloud
* soundcloud
* streamable
* twitch
* twitter
* youtube
* vimeo

I'll add an issue now to better document this.